src/conversions.jl

```diff
@@ -109,18 +109,20 @@ _convert(::Type{Cout}, ::Type{C1}, ::Type{C2}, c) where {Cout<:AbstractRGB,C1<:A
 _convert(::Type{A}, ::Type{C1}, ::Type{C2}, c, alpha=alpha(c)) where {A<:TransparentRGB,C1<:AbstractRGB,C2<:AbstractGray} = (g = convert(eltype(A), gray(c)); A(g, g, g, alpha))
 
 convert(::Type{RGB24},   x::Real) = RGB24(x, x, x)
-convert(::Type{ARGB32},  x::Real) = ARGB32(x, x, x, 1)
+convert(::Type{ARGB32},  x::Real) = ARGB32(x, x, x)
+convert(::Type{ARGB32},  x::Real, alpha) = ARGB32(x, x, x, alpha)
 convert(::Type{Gray24},  x::Real) = Gray24(x)
-convert(::Type{AGray32}, x::Real) = AGray32(x, 1)
+convert(::Type{AGray32}, x::Real) = AGray32(x)
```

Comment:
Probably `AGray32(x, 1)` might be optimized by the compiler, but I think it is better that the alpha completion should be done by the constructors.